XamlControlsGallery/ControlPages/GridViewPage.xaml

```diff
@@ -120,66 +122,31 @@
-                        <Grid.RowDefinitions>
-                            <RowDefinition Height="Auto"></RowDefinition>
-                            <RowDefinition Height="Auto"></RowDefinition>
-                        </Grid.RowDefinitions>
-
-                        <TextBlock x:Name="RowSpaceTextBlock" Grid.Row="0" Grid.ColumnSpan="3" Text="Space Between Rows" Width="150" TextWrapping="Wrap" Margin="0,0,0,5" HorizontalAlignment="Left"/>
-                        <TextBox Grid.Row="1" Grid.Column="0" Width="100" x:Name="RowSpace" PlaceholderText="{x:Bind ActualRowSpace, Mode=TwoWay}"  Margin="0,0,5,10" 
-                                 TextChanged="StyledGrid_ChangeRow" AutomationProperties.LabeledBy="{x:Bind RowSpaceTextBlock}"/>
-                        <RepeatButton x:Name="RowSpacesDec" Grid.Row="1" Grid.Column="1" Content="-" FontSize="15" Width="30" Click="StyledGrid_DecreaseRowSpace" 
-                                Margin="0,0,5,10" VerticalAlignment="Bottom" HorizontalAlignment="Left"/>
-                        <RepeatButton x:Name="RowSpacesInc" Grid.Row="1" Grid.Column="2" Content="+" FontSize="15" Width="30" Click="StyledGrid_IncreaseRowSpace" 
-                                Margin="0,0,0,10" VerticalAlignment="Bottom" HorizontalAlignment="Left"/>
-                    </Grid>
+                    <StackPanel.Resources>
+                        <Style x:Key="DefaultNumberBoxStyle" TargetType="muxc:NumberBox">
+                            <Setter Property="SpinButtonPlacementMode" Value="Inline" />
+                            <Setter Property="StepFrequency" Value="1" />
+                            <Setter Property="HorizontalAlignment" Value="Stretch" />
+                            <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+                            <Setter Property="MaxWidth" Value="250" />
+                            <Setter Property="Margin" Value="0,0,0,16" />
+                        </Style>
+                    </StackPanel.Resources>
 
-                    <Grid HorizontalAlignment="Left">
-                        <Grid.ColumnDefinitions>
-                            <ColumnDefinition Width="Auto"></ColumnDefinition>
-                            <ColumnDefinition Width="Auto"></ColumnDefinition>
-                            <ColumnDefinition Width="Auto"></ColumnDefinition>
-                        </Grid.ColumnDefinitions>
-                        <Grid.RowDefinitions>
-                            <RowDefinition Height="Auto"></RowDefinition>
-                            <RowDefinition Height="Auto"></RowDefinition>
-                        </Grid.RowDefinitions>
+                    <muxc:NumberBox x:Name="ColumnSpace" Minimum="0" Maximum="100" Value="5"
+                                    Header="Space between columns"
+                                    Style="{ThemeResource DefaultNumberBoxStyle}" ValueChanged="NumberBox_ValueChanged">
+                    </muxc:NumberBox>
```

Comment:
nit: Can you make the closing bracket part of the same line rather than separate? So `<muxc:NumberBox x:Name="ColumnSpace" ... ValueChanged="NumberBox_ValueChanged" />`

Reply:
Oh missed that 😅 
Your suggestion definitely looks cleaner!